Apps/W1/External File Storage - SharePoint Connector/app/src/ExtSharePointAccount.Table.al

```diff
@@ -0,0 +1,94 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.ExternalFileStorage;
+
+/// <summary>
+/// Holds the information for all file accounts that are registered via the SharePoint connector
+/// </summary>
+table 4580 "Ext. SharePoint Account"
+{
+    DataClassification = CustomerContent;
+    Caption = 'SharePoint Account';
+
+    fields
+    {
+        field(1; "Id"; Guid)
+        {
```

Comment:
I would suggest to set the property `AllowInCustomizations` to `Never` for the Id, since this is only used internally and shouldn't be exposed to pages by profile customization. (This applies to the other new tables, too).